gcc/rust/backend/rust-compile-expr.cc

```diff
@@ -410,7 +411,8 @@ CompileExpr::visit (HIR::BlockExpr &expr)
   if (!ctx->get_tyctx ()->lookup_type (expr.get_mappings ().get_hirid (),
 				       &block_tyty))
     {
-      rust_error_at (expr.get_locus (), "failed to lookup type of BlockExpr");
+      rust_error_at (expr.get_locus (),
+		     "failed to lookup type of BlockExpr 414");
```

Comment:
What does the 414 means here ? Is it supposed to be
```cpp
rust_error_at (expr.get_locus (), ErrorCode::E0414,
		     "failed to lookup type of BlockExpr");
```
instead ?

Reply:
oops no that was just a random string for grepping, as this error message is duplicated in the compiler. I should have removed it